src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,9 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
-        return false;
+        if (((b >= a)&(b <= c))|((b >= c)&(b <= a)))
```

Comment:
1. Too many `()`. 
2. It's better to use `&&` and `||` for current verifications. Please read about difference between `||` and `|`.
3. Can be just `return ((b >= a)&(b <= c))|((b >= c)&(b <= a));` because result is boolean type.